tsconfig.json

```diff
@@ -1,19 +1,32 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
```

Comment:
Can we keep strict true?

Reply:
The current codebase was not passing the Typescript checker using it, even before, and I don't know Typescript enough to understand the errors.

![2021-11-27-180533_638x690_scrot](https://user-images.githubusercontent.com/1653275/143720772-8b5e2a97-a88b-4d3f-a106-5b3a7d00d852.png)

Reply:
Ok, fixed those, we're `strict` now.